ambry-api/src/main/java/com/github/ambry/config/MySqlNamedBlobDbConfig.java

```diff
@@ -71,7 +70,7 @@ public MySqlNamedBlobDbConfig(VerifiableProperties verifiableProperties) {
     this.localPoolSize = verifiableProperties.getIntInRange(LOCAL_POOL_SIZE, 5, 1, Integer.MAX_VALUE);
     this.remotePoolSize = verifiableProperties.getIntInRange(REMOTE_POOL_SIZE, 1, 1, Integer.MAX_VALUE);
     this.listMaxResults = verifiableProperties.getIntInRange(LIST_MAX_RESULTS, 100, 1, Integer.MAX_VALUE);
-    this.dbTransition = verifiableProperties.getBoolean(DB_TRANSITION, false);
+    this.dbTransition = verifiableProperties.getBoolean(DB_TRANSITION, true);
```

Comment:
is this used somewhere in this PR ? I'd suggest putting this in a separate PR if not used just for easy reverts.

Reply:
Ok, I reverted this one.